tests/HybridRelationsTest.php

```diff
@@ -210,4 +217,63 @@ public function testHybridWith()
                 $this->assertEquals($user->id, $user->books->count());
             });
     }
+
+    public function testHybridMorphToMongoDB(): void
+    {
+        $user = SqlUser::create(['name' => 'John Doe']);
+
+        $photo = Photo::create(['url' => 'http://graph.facebook.com/john.doe/picture']);
+        $photo = $user->photos()->save($photo);
+
+        $this->assertEquals(1, $user->photos->count());
+        $this->assertEquals($photo->id, $user->photos->first()->id);
+
+        $user = SqlUser::find($user->id);
+        $this->assertEquals(1, $user->photos->count());
+        $this->assertEquals($photo->id, $user->photos->first()->id);
+
+        $book = SqlBook::create(['title' => 'Game of Thrones']);
+        $photo = Photo::create(['url' => 'http://graph.facebook.com/gameofthrones/picture']);
+        $book->photo()->save($photo);
+
+        $this->assertNotNull($book->photo);
+        $this->assertEquals($photo->id, $book->photo->id);
+
+        $book = SqlBook::where('title', $book->title)->get()->first();
+        $this->assertNotNull($book->photo);
+        $this->assertEquals($photo->id, $book->photo->id);
+
+        $photo = Photo::first();
+        $this->assertEquals($photo->hasImage->name, $user->name);
+    }
+
+    public function testHybridMorphToSql(): void
+    {
+        $user = User::create(['name' => 'John Doe']);
+
+        $photo = SqlPhoto::create(['url' => 'http://graph.facebook.com/john.doe/picture']);
+        $photo->save();
+        $photo = $user->photos()->save($photo);
+
+        $this->assertEquals(1, $user->photos->count());
```

Comment:
Fails here: 
```
Failed asserting that 0 matches expected 1.
```

But this error seems to come from Laravel itself: if I replace `User` with `SqlUser` and remove the `HybridRelations` trait from every Sql model, it still fails. Event if this is 100% Laravel code.
The relation is stored in database, but the